### ISSUE TITLE
new Dataset object (to apply functions over sets of tables)

### ISSUE THREAD

**Issue by thadguidry** [google/abstracttext#16]
Does it make sense to have representation for Datasets ?
**Join** functions against Datasets(sets of tables(Z200)) ?
REF:  https://schema.org/Dataset

**Comment by vrandezo** [comment]
Yes, absolutely! There should be a type to represent datasets, and functions over that type. That would be pretty awesome.
